### PR TITLE
Change versions for 1.0.0 release

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: com.github.camel-tooling.lsp.eclipse.updatesite/category.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-   <feature url="features/com.github.cameltooling.lsp.eclipse.client.feature_1.0.0.qualifier.jar" id="com.github.cameltooling.lsp.eclipse.client.feature" version="1.0.0.qualifier">
+   <feature url="features/com.github.cameltooling.lsp.eclipse.client.feature_1.0.0.jar" id="com.github.cameltooling.lsp.eclipse.client.feature" version="1.0.0">
```
Comment (lhein) [inline com.github.camel-tooling.lsp.eclipse.updatesite/category.xml]: is that really needed to change? thought it is handled automatically.

File: com.github.cameltooling.lsp.eclipse.client.feature/feature.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <feature
       id="com.github.cameltooling.lsp.eclipse.client.feature"
       label="Tooling for Apache Camel based on Apache Camel implementation of Language Server Protocol"
-      version="1.0.0.qualifier">
```
Comment (lhein) [inline com.github.cameltooling.lsp.eclipse.client.feature/feature.xml]: is that really needed to change? thought it is handled automatically.

File: com.github.camel-tooling.lsp.eclipse.client/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: com.github.camel-tooling.eclipse.client;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.0.0
```
Comment (lhein) [inline com.github.camel-tooling.lsp.eclipse.client/META-INF/MANIFEST.MF]: is that really needed to change? thought it is handled automatically.

File: com.github.camel-tooling.lsp.eclipse.client.tests.integration/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: com.github.camel-tooling.eclipse.client.tests.integration;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.0.0
```
Comment (lhein) [inline com.github.camel-tooling.lsp.eclipse.client.tests.integration/META-INF/MANIFEST.MF]: is that really needed to change? thought it is handled automatically.

File: com.github.cameltooling.lsp.eclipse.client.feature/feature.xml (hunk 2 above)
Comment (apupier) [reply com.github.cameltooling.lsp.eclipse.client.feature/feature.xml]: I don't remember how it is working with tags. Maybe we must even keep it to have the timestamp, will need to investigate

Comment (lhein) [reply com.github.cameltooling.lsp.eclipse.client.feature/feature.xml]: I'd say then we need to go like your initial PR

File: com.github.camel-tooling.lsp.eclipse.updatesite/category.xml (hunk 1 above)
Comment (apupier) [reply com.github.camel-tooling.lsp.eclipse.updatesite/category.xml:3]: yes. see https://github.com/camel-tooling/camel-lsp-client-eclipse/pull/43/files#r200651887

File: com.github.camel-tooling.lsp.eclipse.client/META-INF/MANIFEST.MF (hunk 3 above)
Comment (apupier) [reply com.github.camel-tooling.lsp.eclipse.client/META-INF/MANIFEST.MF:5]: yes, see https://github.com/camel-tooling/camel-lsp-client-eclipse/pull/43/files#r200651887

File: com.github.camel-tooling.lsp.eclipse.client.tests.integration/META-INF/MANIFEST.MF (hunk 4 above)
Comment (apupier) [reply com.github.camel-tooling.lsp.eclipse.client.tests.integration/META-INF/MANIFEST.MF:5]: yes, see https://github.com/camel-tooling/camel-lsp-client-eclipse/pull/43/files#r200651887

**Comment by apupier** [comment]
outdated

**Pull request closed without merging**